paddle/fluid/framework/new_executor/interpretercore.cc

```diff
@@ -122,11 +126,25 @@ paddle::framework::FetchList InterpreterCore::Run(
     ExecuteInstructionList(vec_instruction_);
   }
 
+  if (create_local_scope_) {
+    ClearLocalScope();
+  }
+
   // return Fetch Tensors
   auto* fetch_var = global_scope_->Var(interpreter::kFetchVarName);
   return std::move(*fetch_var->GetMutable<framework::FetchList>());
 }
 
+void InterpreterCore::ClearLocalScope() {
```

Comment:
Rename to  ClearLoDTensorArrayInLocalScope and add some comments

Reply:
done, thx!